about/press.md

```diff
@@ -19,6 +19,7 @@ Any inquiries in regards to press should be sent to info@farsetlabs.org.uk.
   * 12/1/14: [Belfast's Leading tech startups](http://www.theguardian.com/uk-news/2014/jan/12/belfast-leading-tech-startups)- The Guardian
   * 17/1/14: [Technology giant Intel joins forces with Belfast 'hackerspace' Farset Labs](http://www.belfasttelegraph.co.uk/life/technology-gadgets/technology-giant-intel-joins-forces-with-belfast-hackerspace-farset-labs-29926303.html)- Belfast Telegraph
   * 17/1/14: [Intel Joins Farset Labs to Collaborate with the Local Technology Community](http://newsroom.intel.com/community/en_ie/blog/2014/01/17/intel-joins-farset-labs-to-collaborate-with-the-local-technology-community)- Intel Newsroom
+  * 31/4/14 : [Young Programmers Awarded for CoderDojo Creative Bursary Programme](http://nervecentre.org/news/young-programmers-awarded-coderdojo-creative-bursary-programme)
```

Comment:
Everything else in list has a dash followed by the name of the publishing press company, can has in this too?